CandidateReranking.md

```diff
@@ -0,0 +1,368 @@
+Candidate Reranking Service V1
+==============================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+    - [Query](#query)
+        - [Query](#query)
+        - [ParsedInput](#parsed-input)
+        - [ParsedEntity](#parsedentity)
+        - [Keyword](#keyword)
+        - [Relationship](#relationship)
+        - [Entity](#entity)
+        - [Filter](#filter)
+    - [Profile](#profile)
+        - [Profile](#profile)
+        - [RecentJob](#recentjob)
+        - [Skill](#skill)
+- [Response Structure](#response-structure)
+- [Versioning](#versioning)
+
+### Summary
+The Candidate Reranking Service is an HTTP Rest service which performs machine learned ranking on a list of candidate profiles. The service makes use of a versioned reranking-library jar maintained by Textkernel which applies a model trained on Talent Discovery Platform search data.
+
+### Request Structure
+The service supports POST requests to:
+https://api.careerbuilder.com/core/search/candidatereranker
+OAUTH credentials are **required**.
+
+A request is composed of 4 main parts:
+`reranker_config`, `source`, `query`, and `profiles`.
+
+| Param    | Type | Description
+|----------|------|--------|
+| reranker_config | String | **Required**: Accepted Value: **RERANKER_V1** The `reranker_config` determines which learned model to apply to your incoming rerank request. Currently there is only one reranker model to call "RERANKER_V1".
```

Comment:
The double colons on Required/Optional and Accepted Value make me think that Required should be a separate column in this (and subsequent tables) with a boolean value.